### PR TITLE
Connect Infill Polygons AND Multiply Infill

### PR DESCRIPTION
**Pull request by BagelOrb**
frontend setting changes for https://github.com/Ultimaker/CuraEngine/pull/774

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Developers, see issue CURA-5400.

File: resources/definitions/fdmprinter.def.json [hunk 1]
```diff
@@ -1622,7 +1632,18 @@
...
+                    "settable_per_mesh": true
+                },
+                "connect_infill_polygons":
+                {
+                    "label": "Connect Infill Polygons",
+                    "description": "Connect infill paths where they run next to each other. For infill patterns which consist of several closed polygons, enabling this setting greatly reduces the travel time.",
+                    "type": "bool",
+                    "default_value": false,
+                    "value": "infill_pattern == 'cross' or infill_pattern == 'cross_3d' or infill_multiplier % 2 = 0",
+                    "enabled": "infill_pattern == 'concentric_3d' or infill_pattern == 'cross' or infill_pattern == 'cross_3d' or infill_multiplier % 2 == 0",
```
Comment (alekseisasin) [inline resources/definitions/fdmprinter.def.json]: We do not have anymore concentric_3d. It is removed.

Comment (BagelOrb) [reply resources/definitions/fdmprinter.def.json]: fixed: 68c683ee2692c35b46b2173ca3b05f7ce724eab2

**Comment by fieldOfView** [comment]
This is slightly broken. See comment here: https://github.com/Ultimaker/Cura/commit/5cb5e63dcc41d887c9e6bfa49832fae110611238#r29832494

**Comment by BagelOrb** [comment]
Fixed and pushed to master directly: 9c6b676660bca4d2f8e69afb844f4fc35dbd5b97

**Pull request merged**